websites/N/Nix Mangas/presence.ts

```diff
@@ -0,0 +1,27 @@
+const presence = new Presence({
+		clientId: "1073255300263591946",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const { pathname } = window.location,
```

Comment:
`document.location` is preferred in this repository.
```suggestion
	const { pathname } = document.location,
```